src/parser/csql_grammar.y

```diff
@@ -4053,6 +4070,24 @@ only_class_name
 	| user_specified_name
 		{ DBG_TRACE_GRAMMAR(only_class_name, | user_specified_name);
                   $$ = $1; }
+        | class_name_with_server_name
+                {{ DBG_TRACE_GRAMMAR(only_class_name, | class_name_with_server_name);
+
+                        PT_NODE *scs = parser_new_node (this_parser, PT_SPEC);
```

Comment:
only_class_name should return PT_NAME.
Otherwise PT_SPEC will come where PT_NAME was expected to come.

Exception handling is required.

```
csql> alter table t1@cubrid_dblink comment 'The use of dblink is not allowed in the ALTER statement.';

ERROR: before '  comment 'The use of dblink is not allowed in the ALTER state...'
Class _ does not exist.
```

Reply:
I think class_name_with_server_name should not be added to only_class_name, and the parent rule of only_class_name should be added.

e.g.
```
only_class_name_extend
        : only_class_name
        | class_name_with_server_name

only_class_name
        : ONLY user_specified_name
        | user_specified_name
```

Reply:
fixed.